crates/protogen/proto/metastore/catalog.proto

```diff
@@ -163,7 +163,7 @@ message TableEntry {
   options.TableOptions options = 3;
   optional uint32 tunnel_id = 4;
   SourceAccessMode access_mode = 5;
-  repeated options.InternalColumnDefinition columns = 6;
+  optional common.arrow.Schema schema = 6;
```

Comment:
This should remain as internal column definititions (both for backwards compat and we don't want an arrow schema here).